critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -709,6 +721,7 @@ private String getLoggableString(URL url, URI sm) {
         }
         sb.append("...");
         if (sm != null) {
+            downloadAuth = sm.getFragment();
```

Comment:
Don't assign this as a side effect in this (hack) method. Assign independently around line~540.

Now that there is a separate `auth` log property, remove it from the logURL and log the property consistently in FileSyncJob.ERROR messages as well.

Reply:
Added "auth" to line ~540. Updated logURL and added "auth=" to FileSyncJob.ERROR messages.